drivers/display/display_renesas_lcdc.c

```diff
@@ -99,22 +99,28 @@ struct display_smartbond_config {
 	enum display_pixel_format pixel_format;
 };
 
-static inline void lcdc_smartbond_pm_policy_state_lock_get(struct display_smartbond_data *data)
+static inline void lcdc_smartbond_pm_policy_state_lock_get(const struct device *dev)
 {
-#if defined(CONFIG_PM_DEVICE) || defined(CONFIG_PM_DEVICE_RUNTIME)
```

Comment:
Fixed

Reply:
As far as I understood you are not dealing with device power management but system power management. What you doing is putting / getting constraints in the system to avoid the SoC to go standby (which probably will cut off the power for the device).

Reply:
We try to do both. Please check https://github.com/zephyrproject-rtos/zephyr/pull/74009/files#r1639512062.